### DISCUSSION TITLE
Any way to get only the total for a single directory?

### DISCUSSION THREAD

**Discussion by austinbutler** [Byron/dua-cli#239] [Q&A]
The only way I'm able to get it to work is:

```console
dua my_folder | rg total
```

**Comment by Byron** [comment]
I think it's related to the problem that it resolves the given, single path as if it was the current directory.

Thus, `dua $PWD` is not the same as `dua $PWD $PWD`, which is confusing. I remember that the current behaviour was added at some point, but it would certainly be nice to have a way to turn that off. Maybe that would mean that `dua --new-flag` and `dua --new-flag $PWD` would then only list the total for the CWD or the given single directory.

Of course, this wouldn't be necessary if it would treat `dua $PWD` the same like `dua $PWD $PWD`. Maybe this could be changed instead to be consistent. CC @piotrwach .

**Reply by piotrwach** [reply]
@austinbutler Can I ask why do you need this? 

IMHO `dua` strength is in the interactive disk analysis and speed. We should focus on making it absolutely the best in the world in these things things first.

**Reply by Byron** [reply]
Thanks for the excitement :)! I use it way less these days in interactive mode, but every day in `dua` mode.

Besides that, I think the motivation of OP is that `dua $PWD` ought to provide the same result as `dua $PWD $PWD`, but it does not. 
Remembering vaguely, I think this might have been due a problem in the way the paths are handled initially, but then maybe it was made like this intentionally, also for good reason (?). 

I do believe that this should be fixed, and the current behaviour of `dua $PWD` could rather be recreated with something like `dua -C $PWD`, to change CWD before doing anything conveniently.

**Comment by sergeevabc** [comment]
Dua 2.20.3 was the last version that worked the way you describe @austinbutler.

``` console
$ dua folder
 971.76 MiB folder
```

@Byron, how do we get this output now?

**Reply by Byron** [reply]
[This comment still is authoritative](https://github.com/Byron/dua-cli/discussions/239#discussioncomment-9177527).

**Reply by sergeevabc** [reply]
I don't understand that comment.

``` console
$ dua.v2.28.exe -C folder 
error: unexpected argument '-C' found

$ dua.v2.20.3.exe folder
 971.76 MiB folder
````

**Reply by Byron** [reply]
This feature is still aspirational.
